docs/docs/how-tos/tool-calling.md

```diff
@@ -0,0 +1,1008 @@
+# Use tools
```

Comment:
Should we call this "Call tools" instead?

Reply:
Sure!